bip-musig2.mediawiki

```diff
@@ -184,11 +189,13 @@ The following conventions are used, with constants as defined for [https://www.s
 ** ''||'' refers to byte array concatenation.
 ** The function ''x[i:j]'', where ''x'' is a byte array and ''i, j &ge; 0'', returns a ''(j - i)''-byte array with a copy of the ''i''-th byte (inclusive) to the ''j''-th byte (exclusive) of ''x''.
 ** The function ''bytes(n, x)'', where ''x'' is an integer, returns the n-byte encoding of ''x'', most significant byte first.
-** The function ''bytes(P)'', where ''P'' is a point, returns ''bytes(x(P))''.
+** The function ''bytes(P)'', where ''P'' is a point for which ''not is_infinite(P)'', returns ''bytes(x(P))''.
+** The function ''bytes_loose(P)'', where ''P'' is a point, returns ''bytes(P)'' if ''not is_infinite(P)''. Otherwise, it returns ''bytes(32, 0)''.
```

Comment:
How would you feel about using `_extended` instead of `_loose` (motivated by the "extended real number system" which also includes infinity)?

(Just a personal preference since `_loose` sounds kind of strange)

Reply:
Sounds good, done.